app/utilities/protocol_importers/protocol_normalizer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module ProtocolImporters
+  class ProtocolNormalizer
+    def load_all_protocols(_params = {})
+      raise NotImplementedError
+    end
+
+    def load_protocol(_id)
```

Comment:
I wonder if it would be more viable to also pass `params = {}` here of which one parameter can be `id`to account for generality?

Reply:
Hm... `params = {}` or positional arguments like `load_protocol(id:, params: {})`. If we need anything more than ID in the future?

Reply:
Awesome yeah, even better I'm down! :D